ols/app/models/config.py

```diff
@@ -39,6 +39,16 @@ def _dir_check(path: str, desc: str) -> None:
         raise InvalidConfigurationError(f"{desc} '{path}' is not readable")
 
 
+def _file_check(path: str, desc: str) -> None:
+    """Check that path is a readable regular file."""
+    if not os.path.exists(path):
```

Comment:
I think this check is unneccessary for checking whether the path points at a readable file.

Reply:
Resolved